community/6tunnel/fix_help_add_runit.patch

```diff
@@ -0,0 +1,394 @@
+diff --git a/6tunnel.c b/6tunnel.c
+index c33865a..b8d551e 100644
+--- a/6tunnel.c
++++ b/6tunnel.c
```

Comment:
Trailing spaces bothers me too, but patching them in abuilds is quite pointless and undesirable…